src/components/RichTextEditor/utils/parseRawValue.ts

```diff
@@ -21,7 +21,7 @@ export const parseRawValue = (editorId: string, raw?: string): Value => {
     try {
         parsedValue = JSON.parse(raw);
     } catch {
-        const editor = createPlateEditor({ id: editorId, plugins: getEditorConfig() });
+        const editor = createPlateEditor({ plugins: getEditorConfig() });
```

Comment:
Do the tests in clarify work when doing something like this?

```suggestion
        const editor = createPlateEditor({ id: editorId + '_parseRawValue', plugins: getEditorConfig() });
```